src/webui/src/components/Overview.tsx

```diff
@@ -117,7 +114,13 @@ class Overview extends React.Component<{}, OverviewState> {
                         clusterMetaData: clusterMetaData ? clusterMetaData : undefined
                     });
                     // search space format loguniform max and min
-                    const searchSpace = JSON.parse(sessionData.params.searchSpace);
+                    const temp = sessionData.params.searchSpace;
+                    const searchSpace = temp !== undefined
+                        ?
+                        JSON.parse(temp)
+                        :
+                        {}
```

Comment:
Can we put these in one line to make it more concise?

Reply:
Ok